chore: docs moving around (#444)
modernweb-dev/web: docs/blog/introducing-modern-web/index.md

## Guides

> On our all-new [website](https://modern-web.dev), we have a dedicated [Guides](../../guides/index.md) section. It is dedicated in helping you to get confident in our modern web way of working. It feature step by step guides to work with our tools, and we documented common issues developers run into when doing buildless development.

Reviewer: ```suggestion
On our all-new [website](https://modern-web.dev), we have a dedicated [Guides](../../guides/index.md) section. It is meant to help you to get confident building for the modern web. It features step by step guides to work with our tools, and we document common issues developers run into when doing buildless development.
```

In the past years, we've used different frameworks and tools to build our websites. Recently we've become big fans of [11ty](https://www.11ty.dev/). It's simple to use, works with markdown, and generates _just_ plain HTML. There is no runtime javascript involved with displaying page content, making it super fast.

> To add interactivity to our page we started using web components, and applied rollup with our [HTML](https://www.npmjs.com/package/@open-wc/rollup-plugin-html) and [workbox](https://www.npmjs.com/search?q=rollup-plugin-workbox) plugin.

Reviewer: ```suggestion
To add interactivity to our page we started using web components, and applied rollup with our [HTML](https://www.npmjs.com/package/@open-wc/rollup-plugin-html) and [workbox](https://www.npmjs.com/search?q=rollup-plugin-workbox) plugins.
```

Author: `and applied rollup with our html and workbox plugins.`  sounds strange to me - but no clue 😅

Reviewer: Well it's plural :) The original sentence was `and applied rollup with our HTML and workbox plugins to the HTML pages generated by 11ty`. Not great either, so maybe we should rephrase it entirely 🤔

Reviewer: `and added rollup using our HTML and workbox plugins`?

Author: change it to 

To add interactivity to our page we started using web components and applied rollup with our plugins for [HTML](https://www.npmjs.com/package/@open-wc/rollup-plugin-html) and [workbox](https://www.npmjs.com/search?q=rollup-plugin-workbox).

sounds good?

Reviewer: Yeah 👍

## Thanks for reading

> We are incredibly proud of what we archived so far and the direction we are heading and we invite you to join us on our path forward.

Reviewer: ```suggestion
We are incredibly proud of what we have achived so far and the direction we are heading and we invite you to join us on our path forward.
```
